scripts/security-checker.mjs

```diff
@@ -86,12 +88,17 @@ class SecurityChecker {
           const alert = this.alertDictionary[key];
 
           if (alert.type === ALERT_TYPES.dependabot) {
-              const isAlertOpened = await this.isDependabotAlertOpened(alert.number);
+              const matchAlertInIssue = alert.issue.body.match(new RegExp(`\`${this.context.repo}\` - Link:\\s*(https:.*/(dependabot|code-scanning)/(\\d+))`));
+
+              if (!matchAlertInIssue) 
```

Comment:
I don't think that we need it. Each issue should have a corresponding alert. If not there is mistake in the algorithm